tests/src/tracing/eventsource/eventpipeandetw/eventpipeandetw.csproj

```diff
@@ -16,7 +16,12 @@
 
     <!-- Test unsupported outside of windows -->
     <TestUnsupportedOutsideWindows>true</TestUnsupportedOutsideWindows>
-    <DisableProjectBuild Condition="'$(TargetsUnix)' == 'true'">true</DisableProjectBuild>
+
+    <!--
+      Test is blocking official build, disabling it for now.
+      For more information check: #19302, #22045, #22826.
```

Comment:
@jorive  Please make a single issue that then refers to all the other stuff.   This gives you a place to put a proper milestone and any information you collect about it in a way that won't get lost.   It also makes it super clear which update should re-enable this test.    Put that issue number here.